src/backend/gl/src/lib.rs

```diff
@@ -197,6 +264,18 @@ impl PhysicalDevice {
         }
         let name = info.platform_name.renderer.into();
 
+        // Because some drivers don't actually support this stuff, but report
```

Comment:
why not do these checks inside `query_all`?

Reply:
Ok, I've moved it into `query_all`.